apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/DatasetsResourceTest.java

```diff
@@ -92,7 +90,7 @@
 @Testcontainers(parallel = true)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @DisplayName("Dataset Resource Test")
-class DatasetsResourceTest {
+class DatasetsResourceTest extends AbstractDatasetsContainerBaseTest {
```

Comment:
Not clear what is the benefit of creating abstract test classes that are inherited by single implementation?

Reply:
No need, this managed to fix